.github/workflows/update_docs.yml

```diff
@@ -17,16 +17,17 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
+          ref: ${{ github.head_ref }}
           token: ${{ secrets.REPO_SCOPED_TOKEN }}
       - uses: actions/setup-node@v3
         with:
           node-version: 16.x
       - run: npm ci
       - run: ${{ inputs.build_script }}
       - name: Update Documentation
-        uses: EndBug/add-and-commit@v7
+        uses: EndBug/add-and-commit@v9
         with:
           message: "Automated update to repo's documentation from github action"
           add: '*.md'
-          push: true
+          push: origin HEAD
```

Comment:
for my understanding, what is this change for? was there an issue with the `push=true` before?

Reply:
there wasn't a problem before, but now that we bumped to a new major version of `Endbug/add-and-commit`, it had some breaking changes. in particular, it no longer commits to and pushes the PR head branch by default. we have to manually check out the branch (which is why I updated the `actions/checkout` step) and specify the args for the `push` command

Reply:
mm ok, I thought as long as we update the `ref` in `actions/checkout` step, `push=true` still works / would update to the head of whatever branch we check out based on the description they have on their repo. But if it's required to work, then nevermind!

Reply:
based on [here](https://github.com/EndBug/add-and-commit/tree/main#pushing), I thought you needed to pass in a new branch for the default command to work. but, it looks like `new_branch` defaults to an empty string, so the default still works for us. I'll change it back to `push=true`!